### PR TITLE
PP-3426 update content

### PR DESCRIPTION
**Pull request by jonheslop**
Mark and Sonia made some updates to the Create a payment link flow this is the updates

### CODE REVIEW

File: app/controllers/make_a_demo_payment/index_controller.js [hunk 1]
```diff
@@ -33,6 +33,8 @@ module.exports = (req, res) => {
   const currencyMatch = AMOUNT_FORMAT.exec(paymentAmount)
```
Comment (simad) [inline app/controllers/make_a_demo_payment/index_controller.js]: Not a deal breaker, but maybe we could extract this bit in a function, with a name which makes it possible to understand what's it doing, at a glance? 
Just reading the code it's quite hard to understand what's going on - it's padding the amount, but what's the logic behind it? :)

File: app/controllers/payment-links/post-amount-controller.js [hunk 2]
```diff
@@ -31,6 +31,8 @@ module.exports = (req, res) => {
 
   if (formattedPaymentLinkAmount && !currencyMatch[2]) {
```
Comment (simad) [inline app/controllers/payment-links/post-amount-controller.js]: same as above

File: app/controllers/test_with_your_users/submit_controller.js [hunk 3]
```diff
@@ -44,6 +44,8 @@ module.exports = (req, res) => {
   const currencyMatch = AMOUNT_FORMAT.exec(paymentAmount)
```
Comment (simad) [inline app/controllers/test_with_your_users/submit_controller.js]: same. The fact that we had to repeat this function 3 times, makes me think we could actually extract it

Comment (jonheslop) [reply app/controllers/test_with_your_users/submit_controller.js]: Ok I started doing this in this PR but actually I am going to it after I merge, as I have another PR that could benefit by being rebased on top of this

**Pull request merged**